### ISSUE TITLE
Undefined variable e in Kitchen::Driver::Dokken#build_work_image

### ISSUE THREAD

**Issue by esabelhaus** [someara/kitchen-dokken#61]
On a failure, I receive:
```
Failed to complete #create action: [undefined local variable or method `e' for #<Kitchen::Driver::Dokken
```

Did some digging, found this rescue block:
https://github.com/someara/kitchen-dokken/blob/master/lib/kitchen/driver/dokken.rb#L124-L141

it expectes to return an error of e, but the rescue block provides no variable e.

https://github.com/someara/kitchen-dokken/blob/master/lib/kitchen/driver/dokken.rb#L136 must have `=> e` added to correct the issue.

**Comment by esabelhaus** [comment]
Have verified locally by modifying ~/.chefdk/gem contents

**Comment by someara** [comment]
Removed and released as 2.0.3
